### DISCUSSION TITLE
Setup Push and Pull Pipelines using GitLab?

### DISCUSSION THREAD

**Discussion by Jedoe830** [Azure/Enterprise-Scale#824] [Q&A] [answered]
Is it possible to setup the Push and Pull Pipelines in GitLab? Basically, both pipelines just use a docker image. Can we reproduce this in GitLab?

**Comment by jtracey93** [comment, marked as answer]
It absolutely should be possible. As long as you have the pre-reqs on the runner/agent this should work.

Checkout these pipelines here in the AzOps Accelerator for examples of what you'll need to do: https://github.com/Azure/AzOps-Accelerator/tree/main/.github/workflows

Required software (from the top of my head):

- PowerShell Core
- PowerShell Az module
- PowerShell AzOps module
- PowerShell PSFramework module
- JQ

**Comment by powershellwhizz** [comment]
OP did you manage to get this working in Gitlab?
